src/main.c

```diff
@@ -522,7 +526,10 @@ static void get_input(char* in) {
         sweepline(inputwin, 1, 22);
 
         // Finaly print input
-        mvwprintw(inputwin, 1, 22, "%s", in);
+        if (use_interface)
+            mvwprintw_colors(inputwin, 1, 22, COLOR_PAIR_DEFAULT, "%s", in);
+        else
+            mvwprintw(inputwin, 1, 22, "%s", in);
```

Comment:
I think this `if` should be part of the function `mvwprintw_colors` to prevent possible duplicate code.
Same should go for `wprintw_colors`.